modules/ob-defining-system-parameters.adoc

```diff
@@ -0,0 +1,16 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-strategies.adoc
+
+:_content-type: CONCEPT
+[id="ob-defining-system-parameters_{context}"]
+= Defining system parameters
+
+You can use system parameters when defining the steps of a build strategy to access system information, or user-defined information in a `Build` or `BuildRun` CR. You cannot configure or modify system parameters as they are defined at runtime by the `BuildRun` controller.
```

Comment:
I would remove CR after Build and BuildRun, it is recurrent and it makes the paragphs more hard to read. Build and BuildRun is suffisant I think

Reply:
@jkhelil, The OCP writing style guide recommends to specify the object type:
**Use "object", "resource", "custom resource", "spec", etc. as appropriate after the object reference. This helps with clarity and readability.** 

Also, a user does not have time to read a guide in sequential manner. For example, while defining build  strategies, the user might directly jump to this particular module or section and having the object type defined would help him to understand things better.